pkg/defaults/defaults_linux.go

```diff
@@ -130,3 +131,19 @@ func HostsDirs() []string {
 		filepath.Join(xch, "docker/certs.d"),
 	}
 }
+
+// HostGatewayIP returns the non-loop-back host ip if available and returns empty string if running into error.
+func HostGatewayIP() string {
```

Comment:
the `HostGatewayIP` should be the container network interface IP and not from other network interfaces

Reply:
I think by defaulit we are returning the host's real ip now instead of any interface IP. The host ip is accessible from containers.

Reply:
docker are setting the host bridge net interface IP. not sure if the host real IP will fit here, beacuse the host net interface IP that's attached to the container need to be the gateway to route traffic from the container to host.

Reply:
SGTM the real host IP